scripts/presubmit.sh

```diff
@@ -112,9 +112,13 @@ main() {
   if [[ "${run_lint}" -eq 1 ]]; then
     check_cmd golangci-lint \
       'have you installed github.com/golangci/golangci-lint?' || exit 1
+    check_cmd prototool \
```

Comment:
You also could just install on demand with curl or modules https://github.com/uber/prototool/blob/dev/docs/install.md

Reply:
Good idea.